Axial_Descent_Ascent_Operating_Curves_Rotor.m

```diff
@@ -0,0 +1,244 @@
+%  Impulsive theory.
+%  aerodynamic model
+%  \author: Colledà Moreno, Veneruso Salvatore
+%  \version: 1.00
+%
+% Eli-TAARG is free software; you can redistribute it and/or
+% modify it under the terms of the GNU General Public
+% License as published by the Free Software Foundation; either
+% version 3 of the License, or (at your option) any later version.
+%
+% Eli-TAARG is developed by the TAARG Educational organization for
+% educational purposes only.
+% Theoretical and Applied Aerodynamic Research Group - University of Naples Federico II.
+%
+% Eli-TAARG GitHub link: <https://github.com/TAARG-Education/Eli-TAARG>
+%
+% Eli-TAARG is distributed in the hope that it will be useful,
+% but WITHOUT ANY WARRANTY; without even the implied warranty of
+% MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+% General Public License for more details.
+% <http://www.gnu.org/licenses/>.
+%
+% ==============================================================================================
+% |Name        : Axial_Descent_Ascent_Operating_Curves_Rotor.m                           |
+% |Author      : Colledà Moreno, Veneruso Salvatore                                      |
+% |              University of Naples Federico II.                                       |
+% |Version     : 1.01                                                                    |
+% |Date        : 21/12/2020                                                              |
+% |Modified    : 08/01/2021                                                              |
+% |Description : the function plots w(V_infty) and P(V_infty) curves according to
+%                Rotor Simply Impulsive theory.                                          |
+% |Reference   : Renato Tognaccini. Appunti Aerodinamica dell'ala rotante.
+%                Università degli studi di Napoli Federico II. a.a.2020/2021             |
+% |Input       : * the inputs must be Mass of rotorcraft and radius of rotor             |
+% |Output      : w(V_infty) and P(V_infty) plots                                         |
+% |Note        :                                                                         |
+% ==============================================================================================
+function [Power,Induction] =Axial_Descent_Ascent_Operating_Curves_Rotor(M,R,V_inf)
+
+g = 9.81;
+%Altitude Range
+hh    = linspace(0,6000,100);
+
+%Axial Induction
+
+[~, ~, ~, rho1] = atmosisa(hh);
+
+rho = @(h) interp1(hh,rho1,h, 'pchip');
+wh  = @(h) sqrt((M*g)/(2*rho(h)*pi*R^2));
+
+% Non-Dimensional Variables Definition
+V_tilde = @(V,h) V/wh(h);
+
+w_tilde_salita  = @(V,h) (-V_tilde(V,h)/2) + sqrt((V_tilde(V,h)/2)^2+1);
+w_tilde_discesa = @(V,h) (-V_tilde(V,h)/2) - sqrt((V_tilde(V,h)/2)^2-1);
+
+P_tilde_salita   = @(V,h) V_tilde(V,h) + w_tilde_salita(V,h);
+P_tilde_discesa  = @(V,h) V_tilde(V,h) + w_tilde_discesa(V,h);
+
+if nargin==2

```

Comment:
You don't need it here, you can put it lower when it will be really needed.
The less indentations we have, the better it is.